pkg/scheduler/core/generic_scheduler.go

```diff
@@ -704,7 +704,7 @@ func (g *genericScheduler) podFitsOnNode(
 func PrioritizeNodes(
 	ctx context.Context,
 	pod *v1.Pod,
-	nodeNameToInfo map[string]*schedulernodeinfo.NodeInfo,
+	snapshot *nodeinfosnapshot.Snapshot,
```

Comment:
We can make the parameter as `sharedLister schedulerlisters.SharedLister`.

Reply:
```genericScheduler``` is aware of  ```Snapshot```, it is actually the one that updates it, so I think it is a little confusing to use ```Snapshot``` in some part of the code here and ```SharedLister``` in others. I think a potential useful refactoring is to make ```PrioritizeNodes``` a member function of ```genericScheduler```, I can do that in a followup PR.